docs/learn/learn-statemint.md

```diff
@@ -0,0 +1,109 @@
+---
+id: learn-statemint
+title: Statemint
+sidebar_label: Statemint
+description: Statemint and its features
+slug: ../../learn-statemint
+---
+
+Statemint is a generic assets parachain which provides functionality for deploying and transferring
+assets — both Fungible and Non-Fungible Tokens (NFTs). It is a common good parachain on Polkadot
+(not to be confused with [Statemine](https://guide.kusama.network/docs/kusama-statemine/), which is
+a parachain on Polkadot). The native token of Statemint is DOT. Balance transfers and the
+Existential Deposit (ED) of DOT on Statemint are about 1/10th of the values on the Relay chain.
+Apart from the core protocol token DOT, the assets held on Statemint can be broadly categorized as
+
+- Assets backed by an on-chain protocol’s utility
+- Assets with off-chain backing
+- Assets without any backing
+
+## Creating Assets on Statemint
+
+:::info
+
+Before minting assets on Statemint, we recommend that you try out this tutorial on Westmint, which
+is a parachain on Westend. The WND tokens (Westies) are free and are available through a
+[faucet](https://wiki.polkadot.network/docs/learn-DOT#getting-westies).
+
+:::
+
+To create an asset on Statemint, you would need a deposit of 100 DOT. Ensure that your Statemint
+account balance is more than 100 DOT, which should account for the transaction fee as well. You can
```

Comment:
When you follow the UI path below, you also need about 2.1 DOT for the metadata deposit. You can do just asset creation for 100 DOT via the extrinsics tab.

Since most people will probably teleport DOT over, and that takes fees, I'd probably just be safe and tell people to teleport 103 DOT. Then they won't have the frustration of doing 100, then 1 more for fees, then realizing the metadata doesn't work, so doing 2 more.